notificationsStore missing

```
renderer.tsx:25:9 - error TS2339: Property 'notificationsStore' does not exist on type 'typeof import("/xxx/node_modules/@k8slens/extensions/dist/src/extensions/renderer-api/components")'.

25 const { notificationsStore, NotificationStatus } = Renderer.Component;
```

used latest master.

--- a/src/extensions/renderer-api/components.ts
+++ b/src/extensions/renderer-api/components.ts
@@ -19,6 +19,7 @@ import { ConfirmDialog as _ConfirmDialog } from "../../renderer/components/confi
 import type { ConfirmDialogBooleanParams, ConfirmDialogParams, ConfirmDialogProps } from "../../renderer/components/confirm-dialog";
 import openConfirmDialogInjectable from "../../renderer/components/confirm-dialog/open.injectable";
 import confirmInjectable from "../../renderer/components/confirm-dialog/confirm.injectable";
+import notificationsStoreInjectable from "../../renderer/components/notifications/notifications-store.injectable";
 
 // layouts
 export * from "../../renderer/components/layout/main-layout";
@@ -133,3 +134,5 @@ export class TerminalStore {
     console.warn("TerminalStore.resetInstance() does nothing");
   }
 }
+
+export const notificationsStore = asLegacyGlobalForExtensionApi(notificationsStoreInjectable);

--- a/src/renderer/components/notifications/index.ts
+++ b/src/renderer/components/notifications/index.ts
@@ -4,4 +4,3 @@
  */
 
 export * from "./notifications";
-export * from "./notifications.store";

--- a/src/renderer/port-forward/port-forward-notify.tsx
+++ b/src/renderer/port-forward/port-forward-notify.tsx
@@ -5,9 +5,9 @@
 
 import React from "react";
 import { Button } from "../components/button";
-import type { NotificationsStore } from "../components/notifications";
 import { Notifications } from "../components/notifications";
 import type { NavigateToPortForwards } from "../../common/front-end-routing/routes/cluster/network/port-forwards/navigate-to-port-forwards.injectable";
+import type { NotificationsStore } from "../components/notifications/notifications.store";
 
 interface AboutPortForwardingDependencies {
   navigateToPortForwards: NavigateToPortForwards;

